event/src/app/api/users/login/route.ts

```diff
@@ -0,0 +1,69 @@
+import dbConnect from '@/lib/mongo/index';
+import Users from '@/models/User';
+import { NextResponse } from "next/server";
+import bcrypt from "bcrypt";
+import {cookies} from 'next/headers'
+
+import { SignJWT } from 'jose';
+
+
+
+export async function POST(req: Request) {
+    await dbConnect();
+    const body = await req.json();
+    const user = await Users.findOne({ email: body.email }).exec();
+
+    if (!user) {
+        console.log("NO user")
+        return NextResponse.json({ message: "No user by that email..." }, { status: 500 })
+    }
+
+    try {
+        const match = await bcrypt.compare(body.password, user.password);
+        if (match) {
+            //login
+            const payload = JSON.parse(JSON.stringify(user._id))
+            const key = new TextEncoder().encode("encoderKEY")
+        
+            const token = await new SignJWT({
+                userid: payload
+            })
+                .setProtectedHeader({
+                    alg: 'HS256'
+                }) 
+                .setExpirationTime("1hr")
+                .sign(key);
+            
+            
+    
+            cookies().set("accessToken", token);
```

Comment:
Suggest to add `{ secure: true, httpOnly: true }` to improve security. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies

```suggestion
            cookies().set("accessToken", token, { secure: true, httpOnly: true });
```